### PR TITLE
Fix inverted vertical slider

### PR DESCRIPTION
**Pull request by pdecostervgls**
GitHub Issue (If applicable): #

#1350

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?

The value the vertical slider get set to when tapped is inverted.
e.g.: 60% -> 40%, 80% -> 20%

## What is the new behavior?
The vertical slider is set to the value that reflects the position its been tapped at.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
(https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/Slider/Slider.cs [hunk 1]
```diff
@@ -352,7 +352,7 @@ private void OnSliderContainerPressed(object sender, PointerRoutedEventArgs e)
 
 			var newOffset = Orientation == Orientation.Horizontal ?
 				point.X / container.ActualWidth :
-				point.Y / container.ActualHeight;
+				1 - (point.Y / container.ActualHeight);
```
Comment (pdecostervgls) [inline src/Uno.UI/UI/Xaml/Controls/Slider/Slider.cs]: So the easiest way to fix this, is to subtract the result from 1, inverting the inverted value. However, one could argue that the actual bug resides in the `ApplySlideToValue` method.

Comment (davidjohnoliver) [reply src/Uno.UI/UI/Xaml/Controls/Slider/Slider.cs]: Looks fine to me this way, I would prefer to keep the `ApplySlideToValue` logic the same for both horizontal and vertical orientations.

**Comment by pdecostervgls** [comment]
@davidjohnoliver Seems that the UI test failed on wasm. I haven't been able to test using wasm yet.

**Comment by davidjohnoliver** [comment]
Thanks for adding the test :)

**Pull request merged**